src/mixer/state.ts

```diff
@@ -568,6 +568,11 @@ export const setVolume = (
       }
     })
     .on("complete", () => {
+
+      if (level === 0) {
+        volume = -100; // Allow a smooth fade to near silence, then kill it dead.
```

Comment:
You need the converse of this when fading from zero, otherwise we'll have the same problem that we had earlier.